how-to/workspace-platform-starter/client/src/framework/platform/interopbroker.ts

```diff
@@ -665,6 +665,12 @@ export function interopOverride(
 								error
 							);
 						}
+						if (!isEmpty(title)) {
+							// ensure no element tags are provided in the title
+							// we don't know how this information will be used
+							// and title hasn't come from the app directory
+							title = title.replace(/<[^>]*>?/gm, "");
```

Comment:
Could we put this method in the utils.ts as sanitizeString or something like that

Reply:
Done